metrics/notifier.go

```diff
@@ -0,0 +1,31 @@
+package metrics
+
+import "fmt"
+
+// NotifierMetrics is a collection of metrics used in notifier
+type NotifierMetrics struct {
+	SubsMalformed          Meter
+	EventsReceived         Meter
+	EventsMalformed        Meter
+	EventsProcessingFailed Meter
+	SendingFailed          Meter
+	SendersOkMetrics       MetersCollection
+	SendersFailedMetrics   MetersCollection
+}
+
+// ConfigureNotifierMetrics is notifier metrics configurator
+func ConfigureNotifierMetrics(registry Registry, prefix string) *NotifierMetrics {
+	return &NotifierMetrics{
+		SubsMalformed:          registry.NewMeter(metricNameWithPrefix(prefix, "subs.malformed")),
+		EventsReceived:         registry.NewMeter(metricNameWithPrefix(prefix, "events.received")),
+		EventsMalformed:        registry.NewMeter(metricNameWithPrefix(prefix, "events.malformed")),
+		EventsProcessingFailed: registry.NewMeter(metricNameWithPrefix(prefix, "events.failed")),
+		SendingFailed:          registry.NewMeter(metricNameWithPrefix(prefix, "sending.failed")),
+		SendersOkMetrics:       registry.NewMetersCollection(),
+		SendersFailedMetrics:   registry.NewMetersCollection(),
+	}
+}
+
+func metricNameWithPrefix(prefix, metric string) string {
```

Comment:
I think it should be in another file. This function is used in different files like `checker.go` and `notifier.go`. Probably better wil be to move it to `graphite.go`

Reply:
![image](https://user-images.githubusercontent.com/664889/71586549-6c1b9300-2b3c-11ea-92e7-622ee44d3ce5.png)